emulators/bochs/Portfile

```diff
@@ -1,8 +1,8 @@
 PortSystem          1.0
 
 name                bochs
-version             2.6.8
-revision            2
+version             2.6.11
+revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Done